azure_pipelines/verify_msalcpp_per_pr_mac.yml

```diff
@@ -49,9 +49,9 @@ jobs:
      arguments: '--skip-checkout --msal-common-repo $(Agent.BuildDirectory)/CommonCore --remote-ref $(commit_number)'
 
   - task: UsePythonVersion@0
-    displayName: 'Switch to use Xcode 14.1'
+    displayName: 'Switch to use Xcode 15.3'
```

Comment:
Latest macOS14 agent supports xcode 15.4, so in broker we are updating to use 15.4. would you be able to keep it the same ?

Reply:
Broker PR for your reference : https://github.com/AzureAD/azure-activedirectory-tokenbroker-for-objc/pull/1646

Reply:
See comment below

Reply:
Updated